pkg/ucp/aws/ucpcredentialprovider_test.go

```diff
@@ -43,8 +44,11 @@ func (p *mockProvider) Fetch(ctx context.Context, planeName, name string) (*sdk_
 func newMockProvider() *mockProvider {
 	return &mockProvider{
 		fakeCredential: &sdk_cred.AWSCredential{
-			AccessKeyID:     "fakeid",
-			SecretAccessKey: "fakesecretkey",
+			Kind: ucp_datamodel.AWSAccessKeyCredentialKind,
+			AccessKeyCredential: &ucp_datamodel.AWSAccessKeyCredentialProperties{
+				AccessKeyID:     "fakeid",
+				SecretAccessKey: "fakesecretkey",
+			},
```

Comment:
Should we have IRSA unit tests here?

Reply:
The IRSA UT are coming as part of UCP PR for supporting IRSA. (similar to terraform)